contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -936,22 +936,21 @@ Datum getutcdate(PG_FUNCTION_ARGS)
 
 Datum getdate_internal(PG_FUNCTION_ARGS)
 {
-	PG_RETURN_TIMESTAMP(DirectFunctionCall2(timestamp_trunc,CStringGetTextDatum("millisecond"),
-						PointerGetDatum(GetCurrentStatementStartTimestamp())));
+	PG_RETURN_DATUM(DirectFunctionCall1(common_utility_plugin_ptr->timestamptz_datetime, 
```

Comment:
instead of calling it through DirectFunctionCall1, we should refactor original code and then call helper function directly from here. That will save lot of stuff related to security definer while invoking the function